File: Models/Voznja.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;
using System;
namespace Models
{
    public class Voznja{

        [Key]
        public int ID{get;set;}     
        public int IdAutobus{get;set;}  
        public int IdOdredista{get;set;}
        
        public DateTime Vreme{get;set;}
       
        public int ocekivanaDuzinaPuta{get;set;}

        public int DolazniOdlazni { get; set; }

      
        public int BrojPresedanja{get;set;}

        [JsonIgnore]
        public virtual Autobus Autobus{get;set;}
        [JsonIgnore]
        public virtual Odrediste Odrediste{get;set;}
        }
}

File: Client/voznja.js
import { Odrediste } from "./odrediste.js";

export class Voznja {
    constructor(id, idAutobus, idOdredista, vreme, ocekivanaDuzinaPuta, dolazniOdlazni, brojPresedanja) {
      this.id=id;
      this.autobus=idAutobus;
      this.odrediste=idOdredista;
      this.vreme=vreme;
      this.ocekivanaDuzinaPuta=ocekivanaDuzinaPuta;
      this.dolazniOdlazni=dolazniOdlazni;
      this.brojPresedanja=brojPresedanja;
    }
}